pandas/io/formats/format.py

```diff
@@ -545,39 +555,30 @@ def _chk_truncate(self):
         max_cols_adj = self.max_cols_adj
         max_rows_adj = self.max_rows_adj
 
-        truncate_h = max_cols_adj and (len(self.columns) > max_cols_adj)
-        truncate_v = max_rows_adj and (len(self.frame) > max_rows_adj)
-
         frame = self.frame
-        if truncate_h:
-            if max_cols_adj == 0:
-                col_num = len(frame.columns)
```

Comment:
Is this not a valid path?

Reply:
`truncate_h` is defined on L548. if `max_cols_adj==0`, `truncate_h` is `False`?